tests/note-editor.spec.ts

```
@@ -0,0 +1,191 @@
+          parsedLine.json[2][0] &&
+          parsedLine.json[2][0][0]
+        ) {
+          tRPCResultJson = parsedLine // Store this full object
+        } else if (parsedLine.result?.data) {
+          // Sometimes tRPC single responses come directly as {result: {data: ...}}
+          tRPCResultJson = parsedLine
+        }
+      }
+    } catch {
+      // Ignore lines that aren't valid JSON
+      continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)